create_env_file.py

```diff
@@ -20,19 +20,19 @@ def create_env_file(directory_path: Path):
     path_to_env_file: Path = directory_path / destination_file_name
 
     if path_to_env_file.exists():
-        print("✅ Great, you have already an environment file.")
+        print("Great, you have already an environment file.")
```

Comment:
mag das jemand keine emojis :D ?

Reply:
mit emojis:
``
Run python create_env_file.py
Traceback (most recent call last):
  File "D:\a\rebdhuhn\rebdhuhn\create_env_file.py", line 45, in <module>
Uh I see you have no .env file in D:\a\rebdhuhn\rebdhuhn
    create_env_file(directory_path=root_directory_path)
  File "D:\a\rebdhuhn\rebdhuhn\create_env_file.py", line 32, in create_env_file
    print("And we are done.\n\u2139 Please update some credentials for your need, e.g. database credentials.")
  File "C:\hostedtoolcache\windows\Python\3.[12](https://github.com/Hochfrequenz/rebdhuhn/actions/runs/8923348793/job/24507399681#step:5:13).3\x64\Lib\encodings\cp1252.py", line 19, in encode
But do not worry, I have you covered, I try to copy for you the env.example file to.env
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeEncodeError: 'charmap' codec can't encode character '\u2[13](https://github.com/Hochfrequenz/rebdhuhn/actions/runs/8923348793/job/24507399681#step:5:14)9' in position 18: character maps to <undefined>
Error: Process completed with exit code 1.
``

Reply:
@hf-krechan fyi